actions/databases.go

```diff
@@ -0,0 +1,290 @@
+		if err != nil {
+			if aerr, ok := err.(awserr.Error); ok {
+				if aerr.Code() == rds.ErrCodeDBClusterNotFoundFault {
+					log.Println(rds.ErrCodeDBClusterNotFoundFault, aerr.Error())
+					clusterNotFound = true
+				} else {
+					log.Println(aerr.Error())
+				}
+			} else {
+				log.Println(err.Error())
+			}
+		}
+	}
+
+	// search instances for the given db name
+	instancesInput := &rds.DescribeDBInstancesInput{
+		DBInstanceIdentifier: aws.String(c.Param("db")),
+	}
+	instancesOutput, err = rdsClient.Service.DescribeDBInstances(instancesInput)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			if aerr.Code() == rds.ErrCodeDBInstanceNotFoundFault {
+				log.Println(rds.ErrCodeDBInstanceNotFoundFault, aerr.Error())
+				if clusterNotFound {
+					return c.Error(404, aerr)
+				}
+			} else {
+				log.Println(aerr.Error())
+				return c.Error(400, aerr)
+			}
+		} else {
+			log.Println(err.Error())
+			return err
+		}
+	}
+
+	output := struct {
+		*rds.DescribeDBClustersOutput
+		*rds.DescribeDBInstancesOutput
+	}{
+		clustersOutput,
+		instancesOutput,
+	}
+	return c.Render(200, r.JSON(output))
+}
+
+// DatabasesPost creates a database in a given account
+//   It will create a database instance as specified by the `Instance` hash parameters.
+//   If a `Cluster` hash is also given, it will first create an RDS cluster and the instance next.
+func DatabasesPost(c buffalo.Context) error {
+	type DatabaseCreateInput struct {
+		// https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#CreateDBClusterInput
+		Cluster *rds.CreateDBClusterInput
+		// https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#CreateDBInstanceInput
+		Instance *rds.CreateDBInstanceInput
+	}
+	input := DatabaseCreateInput{}
+	if err := c.Bind(&input); err != nil {
+		log.Println(err)
+		return c.Error(400, err)
+	}
+	if input.Instance == nil {
+		return c.Error(400, errors.New("Bad request"))
+	}
+
+	rdsClient := RDS[c.Param("account")]
+
+	// create rds cluster first, if specified
+	if input.Cluster != nil {
+		if input.Cluster.DBSubnetGroupName == nil {
+			input.Cluster.DBSubnetGroupName = aws.String(rdsClient.DefaultSubnet)
+		}
+
+		clusterOutput, err := rdsClient.Service.CreateDBCluster(input.Cluster)
+		if err != nil {
+			if aerr, ok := err.(awserr.Error); ok {
+				log.Println(aerr.Error())
+				return c.Error(400, aerr)
+			}
+			log.Println(err.Error())
+			return err
+		}
+		log.Println("Created RDS cluster", clusterOutput)
+	}
+
+	// create rds instance
+	if input.Instance.DBSubnetGroupName == nil {
+		input.Instance.DBSubnetGroupName = aws.String(rdsClient.DefaultSubnet)
+	}
+
+	instanceOutput, err := rdsClient.Service.CreateDBInstance(input.Instance)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			log.Println(aerr.Error())
+			return c.Error(400, aerr)
+		}
+		log.Println(err.Error())
+		return err
+	}
+	log.Println("Created RDS instance", instanceOutput)
+
+	return c.Render(200, r.JSON(instanceOutput))
+}
```

Comment:
all your returning is the instance creation output?  what about the cluster output?